<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация о книге</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_for_image">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_book_nameS">
       <property name="font">
        <font>
         <pointsize>21</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Название</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_book_name">
       <property name="text">
        <string>Название книги</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_book_authorS">
       <property name="font">
        <font>
         <pointsize>21</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Автор</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_book_author">
       <property name="text">
        <string>Автор</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_book_yearS">
       <property name="font">
        <font>
         <pointsize>21</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Год выпуска</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_book_year">
       <property name="text">
        <string>Год выпуска</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_book_genreS">
       <property name="font">
        <font>
         <pointsize>21</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Жанр</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_book_genre">
       <property name="text">
        <string>Жанр</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
